### ISSUE TITLE
integrate features from freebsd zfsutils (example: do not create snapshot if it has zero size)

### ISSUE THREAD

**Issue by mailinglists35** [zfsonlinux/zfs-auto-snapshot#65]
https://github.com/bdrewery/zfstools/blob/master/lib/zfstools.rb

**Comment by kpande** [comment]
please just use zfstools, since that is ruby, this is bourne shell script.. and the nature of zfs-auto-snapshot is such that I do not think we should continue maintaining it here.

**Comment by mailinglists35** [comment]
@kpande do you mean we should abandon using this package and switch to the ruby version made by @bdrewery?

**Comment by ptx0** [comment]
@mailinglists35 yes, this script is 'unmaintained', hosted only for historical reasons

**Comment by mailinglists35** [comment]
@ptx0 I was under the impression that @highvoltage had taken over (based on this comment https://github.com/zfsonlinux/zfs-auto-snapshot/issues/60#issuecomment-275731433)

**Comment by ptx0** [comment]
@mailinglists35 just another person who can't keep up with the job, i guess. that was a few months ago, still a lot of outstanding issues. I wouldn't hold my breath.

**Comment by kpande** [comment]
https://github.com/zfsonlinux/zfs-auto-snapshot/pull/59

use this PR if you want this functionality. please close this issue.
